playbooks/openstack/openshift-cluster/uninstall.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Unsubscribe RHEL instances
+  hosts: oo_all_hosts
+  become: yes
+  gather_facts: yes
+  tasks:
+  - name: Unsubscribe RHEL instances
+    import_role:
+      name: rhel_subscribe
```

Comment:
Is this the right role? I think it should be rhel_subscribe.

Reply:
whoops - rhel_unsubscribe it is